doc/developers/cython.rst

```diff
@@ -141,3 +141,16 @@ must be ``cimported`` from this module and not from the OpenMP library directly:
 
 The parallel loop, `prange`, is already protected by cython and can be used directly
 from `cython.parallel`.
+
+Types
+~~~~~
+
+Cython code requires to use explicit types. This is one of the reasons to get good
+performance boosts. In order to avoid code duplication, we have a central place
```

Comment:
```suggestion
Cython code requires to use explicit types. This is one of the reasons you get a
performance boost. In order to avoid code duplication, we have a central place
```